specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json

```diff
@@ -1030,6 +1079,95 @@
           }
         }
       }
+    },
+    "/keys/{key-name}/policy": {
+      "get": {
+        "tags": [
+          "Keys"
+        ],
+        "operationId": "GetKeyPolicy",
+        "summary": "Lists the policy for a key.",
+        "description": "The GetKeyPolicy operation returns the specified key policy resources in the specified key vault. This operation requires the keys/get permission.",
+        "parameters": [
+          {
+            "name": "key-name",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the key in a given key vault."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The key policy.",
+            "schema": {
+              "$ref": "#/definitions/KeyPolicy"
+            }
+          },
+          "default": {
+            "description": "Key Vault error response describing why the operation failed.",
+            "schema": {
+              "$ref": "common.json#/definitions/KeyVaultError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "GetKeyPolicy": {
+            "$ref": "./examples/GetKeyPolicy-example.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "keys"
+        ],
+        "operationId": "UpdateKeyPolicy",
+        "summary": "Updates the policy for a key.",
+        "description": "Set specified members in the key policy. Leave others as null. This operation requires the keys/update permission.",
```

Comment:
Sets them as null, or leaves then untouched? There's an important distinction. Similarly for the request model, they should be explicitly null or undefined? Undefined is typical and better (less bandwidth, and common - it's how our SDKs work by default).